Makefile

```diff
@@ -21,7 +21,7 @@ selfcheck: ## check that the Makefile is well-formed
 	@echo "The Makefile is well-formed."
 
 requirements:
-	pip install -q -r requirements.txt --exists-action w
+	pip install -q -r requirements/base.txt --exists-action w
```

Comment:
I think we can make this `quality.txt` so that it installs everything needed for dev.

Reply:
Also below in `quality` we need to change `pep8` with `pycodestyle`.
And actually run the tests in `test` + generate diff-cover. See https://github.com/open-craft/blockstore/blob/master/Makefile#L53

Reply:
Also need to the add the commands to .PHONY.